source/retryable-writes/tests/insertOne-serverErrors.json

```diff
@@ -966,7 +966,10 @@
           ],
           "writeConcernError": {
             "code": 91,
-            "errmsg": "Replication is being shut down"
+            "errmsg": "Replication is being shut down",
+            "errorLabels": [
+              "RetryableWriteError"
```

Comment:
Can you remind me why it is necessary to place the error label inside the write concern error when it is returned as a top-level field on the error object itself?

Reply:
Hm it looks like we do it both ways in these tests; for write concern errors specifically, the error is always embedded (example [here](https://github.com/mongodb/specifications/blob/e0f32a668c514ef5268c92e334ec48ba7b962790/source/retryable-writes/tests/deleteOne-serverErrors.json#L80)), but there are other failpoints where we set the label in the `data` field instead (example [here](https://github.com/mongodb/specifications/blob/e0f32a668c514ef5268c92e334ec48ba7b962790/source/retryable-writes/tests/bulkWrite-errorLabels.json#L34)). I embedded the error here for consistency with other write concern error failpoints, but it might be worth filing another ticket to make the label placement consistent across the board/with the server, which does return the label as a top-level field.

Reply:
Ah okay that makes a lot of sense. Thank you for catching that. I have filed [DRIVERS-1385](https://jira.mongodb.org/browse/DRIVERS-1385) to escalate that issue.